plugins/auth-backend/src/providers/google/provider.ts

```diff
@@ -39,6 +41,19 @@ export class GoogleAuthProvider
     res: express.Response,
     next: express.NextFunction,
   ) {
+    const nonce = crypto.randomBytes(16).toString('base64');
```

Comment:
Need inputs on whether crypto lib is the best way to go about generating a nonce in node

Reply:
That does sound very appropriate as far as i know